v3-nginx/Dockerfile

```diff
@@ -10,6 +10,7 @@ ARG MODSEC_VERSION=3.0.6 \
 
 RUN set -eux; \
     apt-get update; \
+    unset LD_LIBRARY_PATH; \
```

Comment:
What's this for?

Reply:
That solves the build in multi arch. It was failing in debian (probably something that started happening in the last images)... 🤷

Reply:
Ok. I'd put a comment there, describing the issue. Otherwise we'll forget why it's there.

Reply:
Let's remove it and try again.